src/nvim/option.c

```diff
@@ -813,7 +813,7 @@ void set_init_1(void)
  * Set an option to its default value.
  * This does not take care of side effects!
  */
-static void 
+static void
```

Comment:
Do not remove trailing whitespace unless you plan on fixing adjacent code.

Reply:
I will revert it.

Reply:
Done.